run/deployment-previews/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pygithub==1.53
+click==7.1.2
+google-cloud-secret-manager>=2.0.0
```

Comment:
Please either (1) pin this to a specific version or (2) Add an upper bound `<3.0.0`.